### PR TITLE
feat: integrate with hydra cloud

### PR DESCRIPTION
**Pull request by piotrmsc**
## Related Issue or Design Document

 In this PR I have changed hydra-client to ory client in order to support also integration with Ory Cloud Oauth2 Service.

Note: this is a static fake user ;-) 

https://user-images.githubusercontent.com/17701416/187677105-87ab2312-1047-4d9c-858f-41f95a57901b.mp4

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-    baseOptions
-  })
-)
+const configuration = new Configuration({
+  basePath: process.env.HYDRA_ADMIN_URL,
+  baseOptions
+})
+
+if (process.env.ORY_TOKEN) {
+  configuration.accessToken = process.env.ORY_TOKEN
```
Comment (aeneasr) [inline src/config.ts:15]:
```suggestion
  configuration.accessToken = process.env.ORY_PAT
```

**Review by aeneasr — COMMENTED** [review]
Awesome! Could you please also document the new env var in the readme :)

**Review by aeneasr — APPROVED** [review]

**Comment by supercairos** [comment]
Is the `@ory/client` available to Open Source (self-hosted) user or is it juste a Ory Cloud library ?
If so, it is possible to keep two branch:

- One for Ory Cloud users
- One for Open Source (Self Hosted) version ?

**Comment by aeneasr** [comment]
It works for both! :)

**Pull request merged**